docs/data/joy/components/autocomplete/autocomplete.md

```diff
@@ -0,0 +1,394 @@
+---
+product: joy-ui
+title: React Autocomplete component
+githubLabel: 'component: autocomplete'
+waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/combobox/
+---
+
+# Autocomplete
+
+<p class="description">The autocomplete is a text input enhanced by a panel of suggested options when users start typing.</p>
+
+## Introduction
+
+`Autocomplete` is an enhanced version of text input that shows suggested options as the users type and also let them select an option from the list.
+
+{{"demo": "Playground.js", "hideToolbar": true}}
+
+{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+
+## Usage
+
+After [installation](/joy-ui/getting-started/installation/), you can start building with this component using the following basic elements:
+
+```jsx
+import Autocomplete from '@mui/joy/Autocomplete';
+import Input from '@mui/joy/Input';
+
+export default function App() {
+  return <Autocomplete renderInput={(params) => <Input {...params} />} />;
+}
+```
+
+## Basics
+
+The Autocomplete component requires a list of `options` and a `renderInput` function. After triggered by a pointer or keyboard, it displays the provided options. The value must be chosen from a predefined set of allowed values.
+
+{{"demo": "BasicAutocomplete.js"}}
+
+## Anatomy
+
+The Autocomplete component is composed of 8 slots:
```

Comment:
Should we use an ordered list instead of bullet points as we're talking about numbered slots?

Reply:
I don't think so. According to [MDN usage note](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul#usage_notes), the order of the slots has no meaning so it should be bullets. Let me know if my understanding is wrong.